blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/service/ElasticIOConfig.java

```diff
@@ -19,6 +19,7 @@
 public enum ElasticIOConfig implements ConfigDefaults {
 
     ELASTICSEARCH_HOSTS("127.0.0.1:9300"),
```

Comment:
You could remove `ELASTICSEARCH_HOSTS` from here and `ConfigDefaults` to avoid any accidental attempts later to use the transport connection.